boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -399,15 +399,10 @@ arduino_serial: &flexcomm12 {
 };
 
 &dma0 {
-	/*
-	 * The total number of dma channels available is defined by
-	 * FSL_FEATURE_DMA_NUMBER_OF_CHANNELS in the SoC features file.
-	 * Since memory from the heap pool is allocated based on the number
-	 * of DMA channels, set this property to as many channels is needed
-	 * for the platform. Adjust HEAP_MEM_POOL_SIZE in case you need more
-	 * memory.
-	 */
-	dma-channels = <37>;
+	status = "okay";
+};
+
+&dma1 {
```

Comment:
Why are you enabling `dma1` for the RT595 and RT685 here, if it was not previously enabled?

Reply:
why not

Reply:
Are we using this DMA node anywhere? If not then we are needlessly increasing the size of the build for the RT595 EVK